examples/testmodule/plugins/submod.py

```diff
@@ -5,3 +5,17 @@
 """
 
 submod_loaded: bool = True
+
+
+MODULE_CACHE_ONE = set()
+MODULE_CACHE_TWO = set()
+
+
+def inmanta_reset_state_one() -> None:
+    global MODULE_CACHE_ONE
+    MODULE_CACHE_ONE = set()
+
+
+def inmanta_reset_state_two() -> None:
+    global MODULE_CACHE_TWO
+    MODULE_CACHE_TWO = set()
```

Comment:
Why support multiple reset methods?

Reply:
1. I thought it would add more flexibility in case of complex cleanup
2. It doesn't really add complexity to support it
If you'd prefer just a single function, that would be fine by me as well.

Reply:
We can keep it like this. I just don't see the use case, but that doesn't meant there isn't any.